utilities/github-sarif-helper/src/semgrep-json-to-sarif.py

```diff
@@ -9,19 +9,127 @@ def build_sarif_result_locations(finding):
                 "uriBaseId": "%SRCROOT%"
             },
             "region": {
-                "endColumn": finding['end']['col'] if (finding['end']['col'] > 0) else 1,
+                "endColumn": parse_loc_col(finding['end']),
                 "endLine": finding['end']['line'],
                 "snippet": {
                     "text": finding['extra']['lines']
                 },
-                "startColumn": finding['start']['col'] if (finding['start']['col'] > 0) else 1,
+                "startColumn": parse_loc_col(finding['start']),
                 "startLine": finding['start']['line']
             }
         }
     }]
 
+def parse_loc_col(loc):
+    return loc['col'] if (loc['col'] > 0) else 1
+
+def parse_loc_cliloc(trace_item):
+    loc_data = trace_item[1][0]
+    return {
+        "location": {
+            "physicalLocation": {
+                "artifactLocation": {
+                    "uri": loc_data['path'],
+                },
+                "region": {
+                    "endColumn": parse_loc_col(loc_data['end']),
+                    "endLine": loc_data['end']['line'],
+                    "startColumn": parse_loc_col(loc_data['start']),
+                    "startLine": loc_data['start']['line'],
+                    "snippet": { "text": trace_item[1][1] }
+                }
+            }
+        },
+        "nestingLevel": 0
+    }
+
+def parse_loc_no_label(trace_item):
+    loc_data = trace_item[0]
+    return {
+        "location": {
+            "physicalLocation": {
+                "artifactLocation": {
+                    "uri": loc_data['path'],
+                },
+                "region": {
+                    "endColumn": parse_loc_col(loc_data['end']),
+                    "endLine": loc_data['end']['line'],
+                    "startColumn": parse_loc_col(loc_data['start']),
+                    "startLine": loc_data['start']['line'],
+                    "snippet": { "text": trace_item[1] }
+                }
+            }
+        },
+        "nestingLevel": 0
+    }
+
+def parse_loc_content(trace_item):
+    loc_data = trace_item[0]['location']
+    return {
+        "location": {
+            "physicalLocation": {
+                "artifactLocation": {
+                    "uri": loc_data['path'],
+                },
+                "region": {
+                    "endColumn": parse_loc_col(loc_data['end']),
+                    "endLine": loc_data['end']['line'],
+                    "startColumn": parse_loc_col(loc_data['start']),
+                    "startLine": loc_data['start']['line'],
+                    "snippet": { "text": trace_item[0]['content'] }
+                }
+            }
+        },
+        "nestingLevel": 0
+    }
+
+def parse_loc_clicall(trace_item):
+    return {}
+
```

Comment:
```suggestion
```

Unless I'm missing a call to this somewhere (guessing left over from an earlier iteration?)